pkg/inspect/inspect.go

```diff
@@ -14,14 +15,25 @@ const (
 	REALITY_SOURCE
 )
 
+type LaunchableVersion struct {
+	Location string                    `json:"location,omitempty"`
```

Comment:
Based on my grep skills, this type extension thing is done in exactly one place. If we expect LaunchableVersion to replace all need to reference Location (we do) then I'm ok with this sort of code debt.